.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+<!--- Describe your changes in detail -->
+
+
+## Context
+<!--- What problem does this change solve? -->
+<!--- Include an issue number, if the issue this PR solves has been logged previously  -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds new functionality)
+- [ ] Breaking change (fix/feature that changes existing functionality)
+
+## PR Checklist:
```

Comment:
This is a nifty idea; might be nice to add a "all new features have been documented" option here, too.